src/divert.h

```diff
@@ -5,6 +5,10 @@
 #ifndef _EMONESP_DIVERT_H
 #define _EMONESP_DIVERT_H
 
+#define _DIVERT_TYPE_UNSET -1
+#define _DIVERT_TYPE_SOLAR   0
+#define _DIVERT_TYPE_GRID  1
```

Comment:
Please use an enum rather than #define, similar to [EventType](https://github.com/OpenEVSE/ESP32_WiFi_V4.x/blob/master/src/event_log.h#L19)